pyproject.toml

```diff
@@ -44,7 +44,7 @@ docstring-code-format = true
 docstring-code-line-length = "dynamic"
 
 [tool.ruff.lint]
-select = ["D", "F401", "I"]
+select = ["A", "D", "F401", "I"]
```

Comment:
I should probably RTFM, but what's "A" here? I know the other part of this list are already here, but would it make sense to add a comment about what these are?

Reply:
https://docs.astral.sh/ruff/rules/#flake8-builtins-a

I'll add some comments to help out the next person.